Build/docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
 
     typo3:
         build:
-            context: .
+            context: ..
```

Comment:
It's true that the referenced Dockerfile does not exist. However the context path is correct.